### PR TITLE
[NCL-7284] Implement top bar announcement messages

### PR DESCRIPTION
**Pull request by patrikk0123**

### CODE REVIEW

**Comment by patrikk0123** [comment]
This issue will require integration with TopBar component.
Also, maybe Context API will be required to immediately propagate newly created announcement to the TopBar.

**Comment by patrikk0123** [comment]
Hi, thanks for feedback:

points no. 1 - 3:
This is solved by TopBar component, which will be used here when https://github.com/project-ncl/pnc-web-ui-react/pull/89 is merged.

points no. 4 - 5:
I already created separate Jira for this which will integrate websockets, but websocket service must be implemented first.
This means for now only announcement creation will be possible and announcement will be displayed just when you refresh page.

**Comment by DnsZhou** [comment]
Hi @patrikk0123 , That sounds good and I will just approve this PR.

Thanks and regards,
- Dennis

**Review by DnsZhou — APPROVED** [review]

File: src/components/AdministrationPage/AdministrationPage.tsx [hunk 1]
```diff
@@ -125,29 +147,45 @@ export const AdministrationPage = () => {
...
-                      <div style={maintenanceSwitchStyle}>
-                        <Switch
-                          id="form-maintenance-mode-switch"
-                          name="form-maintenance-mode-switch"
-                          label="Maintenance Mode On"
-                          labelOff="Maintenance Mode Off"
-                          isChecked={isMaintenanceModeOn}
-                          onChange={() => {
-                            setIsMaintenanceModeOn(!isMaintenanceModeOn);
+                  <ServiceContainerCreating {...dataContainerAnnouncement} title="Announcement">
```
Comment (matedo1) [inline src/components/AdministrationPage/AdministrationPage.tsx:150]: This PR needs to be rebased after patch PR is merged

**Review by matedo1 — APPROVED** [review]

**Pull request merged**